app/controllers/HomeController.php

```diff
@@ -1,23 +1,26 @@
 <?php
 
-class HomeController extends BaseController {
+class HomeController extends BaseController
+{
 
-	/*
-	|--------------------------------------------------------------------------
-	| Default Home Controller
-	|--------------------------------------------------------------------------
-	|
-	| You may wish to use controllers instead of, or in addition to, Closure
-	| based routes. That's great! Here is an example controller method to
-	| get you started. To route to this controller, just add the route:
-	|
-	|	Route::get('/', 'HomeController@showWelcome');
-	|
-	*/
-
-	public function showWelcome()
+	public function showIndex()
 	{
-		return View::make('hello');
+		$phoneNumber	 = false;
+		$errorMessage	 = false;
+
+		if (Input::has('country')) {
+			$twilioClient = new Services_Twilio(getenv('TWILIO_API_SID'), getenv('TWILIO_API_TOKEN'));
+
+			$numberProvider = new \models\PhoneNumberProvider($twilioClient, Input::get('country'));
```

Comment:
Why `PhoneNumberProvider` in models namespace?
What  is core idea of `PhoneNumberProvider`? 
Why don't use Twilio service provider?